README.md

```diff
@@ -1,24 +1,59 @@
-# 👾 Processbot
+# 👾 processbot
 
 # Commands
 
 To be posted as a comment in the Pull Request. **Your whole comment should only have the command**.
 
-- `bot merge` to automatically merge it once checks pass (if approvals have been
-  given).
-- `bot merge force` to attempt merge without waiting for checks (if approvals
-  have been given).
-- `bot merge cancel` to cancel a pending `bot merge`.
-- `bot compare substrate` to see a diff between current branch's Substrate.
+- `bot merge`: [if approved](#criteria-for-merge), merge once checks pass.
+- `bot merge force`: [if approved](#criteria-for-merge), merge immediately
+  while disregarding checks.
+- `bot merge cancel`: cancel a pending `bot merge`.
+- `bot compare substrate`: see a diff between current branch's Substrate
```

Comment:
is `substrate` substantial? How does it work from pokadot? Maybe because the feature is rather unique between these two projects it'd make sense to name it `companion/coupled/related`?

Reply:
I took a look: it's specifically hardcoded for Polkadot and Substrate.

https://github.com/paritytech/parity-processbot/blob/89c83854d014afa9accc1581962745f8d751600c/src/constants.rs#L6

Implementation: https://github.com/paritytech/parity-processbot/blob/66db28dab94f7fdc3acaf7f9b199fae44aa570ba/src/webhook.rs#L632

TBH I have never used this command nor have seen anyone using it (it's from the previous maintainers).  I'm not sure what it does. I'd rather delete it if it's not being used.